src/styles/pages/services/services.module.scss

```diff
@@ -0,0 +1,178 @@
+.mainContainer {
+    font-family: Arial; //import later 
+    display: flex;
+    flex-direction: column;
+}
+
+.servicesContainer {
+    background-color: var(--color-white);
+    color: var(--color-maroon);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 32px;
+    row-gap: 40px;
+    justify-content: space-around;
+
+    .pageHeading {
+        font-size: 3.4375rem;
+        //font-family: "Be Vietnam";
+    }
+}
+
+.boxContainer {
+    display: flex;
+    flex-direction: row;
+    column-gap: 40px;
+    justify-content: center;
+
+    .serviceBox {
+        background-color: var(--color-beige);
+        border-radius: 12px;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .topServiceBox {
+        row-gap: 65px;
+        width: 402px;
+        height: 300px;
+
+
+        .boxHeadingTop {
+            font-size: 2.8125rem;
+            text-align: center;
+            padding-top: 40px;
+        }
+
+        >p {
+            font-size: 1.875rem;
+            color: black;
+            font-weight: bold;
+            text-align: center;
+        }
+    }
+
+    .bottomServiceBox {
+        width: 492px;
+        height: 338px;
+        row-gap: 26px;
+        justify-content: center;
+
+        >h2 {
+            text-align: center;
+            font-size: 2.5rem;
+        }
+
+        .desc {
+            font-size: 2rem;
+            color: black;
+            list-style-position: inside;
+            padding-left: 27px;
+        }
+
+        .translatorsList {
+            columns: 2;
+        }
+    }
+}
+
+
+// Slider code 
+.slideContainer {
+    background-color: var(--color-white);
+    display: flex;
+    flex-direction: column;
+    color: var(--color-gold);
+}
+
+.tabContainer {
+    display: flex;
+    flex-direction: row;
+    column-gap: 28px;
+}
+
+.tab {
+    width: 300px;
```

Comment:
here, to make the tabs go full width like in the figma file replace width with 25% (not sure if i had said to remove it the first time around. If so, my apologies!)